sunpy/instr/goes.py

```diff
@@ -147,13 +160,13 @@ def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH
     2014-01-01 00:00:06  7e-07  7e-06  11.28295376  4.78577516e+48
 
     """
-
     # Check that input argument is of correct type
     if not isinstance(goeslc, lightcurve.GOESLightCurve):
         raise TypeError("goeslc must be a GOESLightCurve object.")
```

Comment:
Could the error give a better message than `"goeslc .."`, goeslc is the internal variable name, the user may not know what it refers.  Maybe change it for `"input"`.  I've looked up if there's a better way to do it, but I've not found it yet..